### PR TITLE
Update blacklight-marc to 8.0.0

### PR DESCRIPTION
**Pull request by sensei100**
It looks like this module was renamed about six months ago.  Both the old and new names work for me locally, but I am hoping updating it will fix the issue on QA that happened yesterday.  The error showed:

uninitialized constant Blacklight::Solr::Document::Marc
Did you mean? Blacklight::Marc

### CODE REVIEW

**Comment by dkinzer** [comment]
I don't think this is correct.  In the blacklight-marc documentation they still say that to use you need to extend using `Blacklight::Solr::Document::Marc` and they have a spec that is working: https://github.com/projectblacklight/blacklight-marc/blob/main/spec/controllers/catalog_controller_spec.rb#L5-L7

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ class SolrDocument
   SolrDocument.use_extension(Blacklight::Document::Sms)
 
   # Add Blacklight-Marc extension:
-  SolrDocument.use_extension(Blacklight::Solr::Document::Marc) do |doc|
```
Comment (dkinzer) [inline app/models/solr_document.rb:38]: I don't think this is correct. In the blacklight-marc documentation they still say that to use you need to extend using Blacklight::Solr::Document::Marc and they have a spec that is working: https://github.com/projectblacklight/blacklight-marc/blob/main/spec/controllers/catalog_controller_spec.rb#L5-L7

Comment (sensei100) [reply app/models/solr_document.rb:38]: @dkinzer that worked for me locally, but that is what is throwing an error on QA.  The actual model file has an example using this configuration: https://github.com/projectblacklight/blacklight-marc/blob/main/app/models/concerns/blacklight/marc/document_extension.rb#L24

I don't think they updated the readME.

Comment (dkinzer) [reply app/models/solr_document.rb:38]: @sensei100 Yup. That makes sense

**Review by dkinzer — APPROVED** [review]

**Pull request merged**